app/src/main/java/com/example/happymeals/recipe/RecipeStorageActivity.java

```diff
@@ -29,7 +29,7 @@ protected void onCreate(Bundle savedInstanceState) {
         recipeListView = findViewById(R.id.recipe_list);
 
         recipeListView = (ListView) findViewById(R.id.recipe_list);
-        //recipeStorage = new RecipeStorage();
+        recipeStorage = new RecipeStorage(fsm);
```

Comment:
I believe this should be `recipeStorage = RecipeStorage.getInstance();` unless that wasn't working for you